From 07bd99ba9ae89b31bba161bd1e711855d1864b5b Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Mon, 9 Sep 2013 15:22:20 +0200
Subject: [PATCH] sync parameter values with keystone-dist.conf

---
 etc/keystone.conf.sample | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/etc/keystone.conf.sample b/etc/keystone.conf.sample
index c45c84b..735505c 100644
--- a/etc/keystone.conf.sample
+++ b/etc/keystone.conf.sample
@@ -280,7 +280,7 @@
 
 # The messaging driver to use, defaults to rabbit. Other
 # drivers include qpid and zmq. (string value)
-#rpc_backend=rabbit
+#rpc_backend=qpid
 
 # The default exchange under which topics are scoped. May be
 # overridden by an exchange name specified in the
@@ -412,7 +412,7 @@
 # (Optional) Name of log file to output to. If no default is
 # set, logging will go to stdout. (string value)
 # Deprecated group/name - [DEFAULT]/logfile
-#log_file=<None>
+#log_file=/var/log/keystone/keystone.log
 
 # (Optional) The base directory used for relative --log-file
 # paths (string value)
@@ -551,7 +551,7 @@
 
 # Catalog template file name for use with the template catalog
 # backend. (string value)
-#template_file=default_catalog.templates
+#template_file=/etc/keystone/default_catalog.templates
 
 # Keystone catalog backend driver (string value)
 #driver=keystone.catalog.backends.sql.Catalog
@@ -591,7 +591,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=sqlite:////keystone/openstack/common/db/$sqlite_db
+#connection=mysql://keystone:keystone@localhost/keystone
 
 # The SQLAlchemy connection string used to connect to the
 # slave database (string value)
@@ -657,7 +657,7 @@
 #
 
 # Keystone EC2Credential backend driver (string value)
-#driver=keystone.contrib.ec2.backends.kvs.Ec2
+#driver=keystone.contrib.ec2.backends.sql.Ec2
 
 
 [endpoint_filter]
@@ -1066,7 +1066,7 @@
 
 # Name of the paste configuration file that defines the
 # available pipelines (string value)
-#config_file=keystone-paste.ini
+#config_file=/usr/share/keystone/keystone-dist-paste.ini
 
 
 [policy]
